src/utils/sphereIntersection.h

```diff
@@ -0,0 +1,16 @@
+#ifndef sphereIntersection_H

```

Comment:
Which one is better? the guard or pragma once?

Reply:
I think "Pragma Once" is generally better (I just found out about it). But if portability across all compilers is important "include guard" might be a better choice. 
I've changed the header file to use #Pragma once